models/articles.js

```diff
@@ -41,3 +41,30 @@ exports.selectArticleComments = (article_id) => {
       return rows;
     });
 };
+
+exports.insertComment = (article_id, newcomment) => {
+  const { author, body } = newcomment;
+  const values = [article_id, author, body];
+
+  if (typeof body !== "string" || body.length <= 1) {
+    return Promise.reject({ status: 400, message: "Bad request" });
+  }
+
+  if (typeof author !== "string" || author.length <= 1) {

```

Comment:
is this a duplicate? If so please remove

Reply:
Thanks Alex, this one check the body in an if statment and the author in another